oci/oci.go

```diff
@@ -246,6 +246,13 @@ func (r *Runtime) CreateContainer(c *Container, cgroupParent string) error {
 		return err
 	}
 
+	// We will delete all container resources if creation fails
+	defer func() {
+		if err != nil {
```

Comment:
Isn't it safer to name the error to the function and then check  that exact error? Then you would not need to do the changes below.

Reply:
Yes, safer and cleaner. Thanks.